packages/common/src/utils/array.ts

```diff
@@ -81,6 +121,22 @@ export function orderBy<T>(
   });
 }
 
+/**
+ * Returns true if arrays are equal
```

Comment:
What do you think about using "Checks" instead of "Returns true" for this and other cases?

Reply:
Yeah that's a good idea,
Refactored!